api/src/main/java/org/openmrs/module/eptsreports/reporting/calculation/pvls/RoutineCalculation.java

```diff
@@ -275,13 +275,14 @@ private boolean isOnRoutineCriteria1(
     for (Obs vlObs : viralLoadForPatientTakenWithin12Months) {
       if (vlObs != null && vlObs.getObsDatetime() != null) {
         Date vlDate = vlObs.getObsDatetime();
+        float monthsSince =
+            EptsCalculationUtils.monthsSinceIncludingDaysDiff(vlDate, artInitiationDate);
         if (criteria.equals(PatientsOnRoutineEnum.ADULTCHILDREN)
-            && (EptsCalculationUtils.monthsSince(artInitiationDate, vlDate) > 6
-                && EptsCalculationUtils.monthsSince(artInitiationDate, vlDate) <= 9)) {
+            && (monthsSince > 6 && monthsSince <= 9)) {
           return true;
         } else if (criteria.equals(PatientsOnRoutineEnum.BREASTFEEDINGPREGNANT)
-            && EptsCalculationUtils.monthsSince(vlDate, artInitiationDate) > 3
-            && EptsCalculationUtils.monthsSince(vlDate, artInitiationDate) <= 6) {
+            && monthsSince > 3
```

Comment:
Still we NOT including exactly 3 months, by doing  monthsSince >= 3? I think FGH does that, but Our specs exclude the exact match, otherwise all looks good

Reply:
I am confirming with Pinki :+1:

Reply:
Pinki says > 6 is fine, we can leave it.